manifests/07_deployment.yaml

```diff
@@ -59,15 +56,3 @@ spec:
         requests:
           cpu: 10m
           memory: 20Mi
-    volumes:
-      - name: cacert
-        # Extract ca-bundle.pem to /usr/share/pki/ca-trust-source if present.
-        # Let the pod start when the ConfigMap does not exist or the certificate
-        # is not preset there. The certificate file will be created once the
-        # ConfigMap is created / the cerificate is added to it.
-        configMap:
-          name: cloud-provider-config
-          items:
-            - key: ca-bundle.pem
-              path: ca-bundle.pem
-          optional: true
```

Comment:
From commit message:

>  if we try to mount the cacert volume, it fails.

How can it fail if it's `optional: true`?

Reply:
The error message is

> The Deployment "openstack-cinder-csi-driver-operator" is invalid: spec.template.spec.containers[0].volumeMounts[0].name: Not found: "cacert"